setup.py

```diff
@@ -41,7 +41,7 @@ def run_tests(self):
     version = re.compile(r".*__version__ = '(.*?)'",
                          re.S).match(v_file.read()).group(1)
 
-requires = ['setuptools', 'pyyaml', 'click>=3.1, <6.7', 'requests', 'IPy']
+requires = ['setuptools', 'pyyaml', 'click>=3.1, >=6.7', 'requests', 'IPy']
 
 tests_require = ['pytest', 'pytest-cov', 'tox']
 if sys.version_info < (2, 7):
```

Comment:
With  `Click >= 7.0` `python 2.6` is deprecated so this should not be needed anymore  
supported version are: `python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",`